renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+    "branchPrefix": "renovate-",
+    "prHourlyLimit": 0,
+    "lockFileMaintenance": {
```

Comment:
I don't think we'll be utilizing lock files in this repo, so this could be removed.